simple_ai_agents/arxiv_researcher_agent_with_memori/.env.example

```
@@ -0,0 +1,4 @@
+NEBIUS_API_KEY="your nebius-api-key-here"  # Replace with your actual Nebius AI API key
+TAVILY_API_KEY="your-tavily-api-key-here"  # Replace with your actual Tavily API key
+EXAMPLE_MODEL_NAME ="moonshotai/Kimi-K2-Instruct" # Replace with any other supported models
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXAMPLE_MODEL_NAME  key should go before the NEBIUS_API_KEY key

(UnorderedKey)